packages/@vue/cli-plugin-router/generator/template/src/router/index.js

```diff
@@ -26,6 +32,8 @@ const routes = [
   }
 ]
 
+Vue.use(VueRouter)
```

Comment:
Thanks for the update!
One last small suggestion here: it might be easier to read if we put the `Vue.use` statement back to the beginning of the file, right after the `import` statements.

Reply:
Done, `Vue.use` is in place